genie_registry/vcf.py

```diff
@@ -215,3 +174,101 @@ def _validate(self, vcfdf):
         # will parse a VCF in a detailed way,
         # and output with warnings or errors if the format is not adhered too
         return total_error, warning
+
+    def validate_tumor_and_normal_sample_columns_exist(
```

Comment:
rename to something more appropriate like `validate_tumor_and_normal_sample_columns`